test/performance/kueue/prerequisites/prometheus-sm-crd.yaml

```diff
@@ -0,0 +1,685 @@
+# https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/v0.60.1/example/prometheus-operator-crd/monitoring.coreos.com_servicemonitors.yaml
+---
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Can you put this as installation instructions? We shouldn't need to copy this file into the kueue repo.

Reply:
We do not need to pre-provision this CRD if we use https://github.com/kubernetes-sigs/kueue/pull/462/commits/ca3bf8ff9fd4731ba17b1de87ed57b7bdf60ce63

Please see my comment in the next message.